packages/blade/src/components/Input/_decisions/decisions.md

```diff
@@ -0,0 +1,295 @@
+| errorText                                                                                         | `string`                          | No       |                                                           | Shown when the `validationState` of the input field is set to Error. Only one of `helpText`, `errorText` or `successText` is shown at a time in the priority order as `errorText`, `successText`, `helpText`                                                                          |
+| successText                                                                                       | `string`                          | No       |                                                           | Shown when the `validationState` of the input field is set to Success. Only one of `helpText`, `errorText` or `successText` is shown at a time in the priority order as `errorText`, `successText`, `helpText`                                                                        |
+| autoFocus                                                                                         | `boolean`                         | No       | `false`                                                   | The autofocus global attribute is a Boolean attribute indicating that an element should be focused on page load. [Web Reference](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/autofocus), [Native Reference](https://reactnative.dev/docs/textinput#autofocus) |
+| autofillSuggestionType                                                                            | `password`, `passwordNew`, `none` | No       |                                                           | Specifies autocomplete hints for the system, so it can provide autofill                                                                                                                                                                                                               |
+| keyboardReturnKeyType [Check this table for full reference](#return-key-types-for-web-and-native) | `go`, `done`, `next`, `send`      | No       | Closest based on the `type` attribute                     | Determines how the return key should look on the keyboard on mobile devices or virtual keyboard                                                                                                                                                                                       |
+
+## Notes
+
+Reusable components candidate
+
+- label
+- helpText/errorText/successText
+- character counter(only input field)
+- clear icon button
+- prefix/suffix
+
+## Open questions
+
+- [ ] Do we need to expose `autoCapitalize` for native?
```

Comment:
I think it can be useful because the default is capitalize first letter of each sentence

Reply:
It should be added since native already provides a prop for it without need to calculate the same in JS to avoid jitter